### DISCUSSION TITLE
Database connection

### DISCUSSION THREAD

**Discussion by kevinChnn** [php4dev/heroku-wordpress#42] [Q&A] [answered]
Hi, I follow the Installation Guide but the app always returns HTTP ERROR 500.
It might be a database problem. Do I miss something about the database setup?

**Comment by kevinChnn** [comment, marked as answer]
Sorry for duplicate question as #41

**Comment by javadev** [comment]
Please confirm the postgres database version.

**Reply by kevinChnn** [reply]
Postgresql 13

**Comment by javadev** [comment]
We need a fix for this script:

CREATE TABLE $wpdb->options (
	option_id bigint(20) unsigned NOT NULL auto_increment,
	option_name varchar(191) NOT NULL default '',
	option_value longtext NOT NULL,
	autoload varchar(20) NOT NULL default 'yes',
	PRIMARY KEY  (option_id),
	UNIQUE KEY option_name (option_name),
	KEY autoload (autoload)
) $charset_collate;

**Reply by kevinChnn** [reply]
Excuse me, which script are you referring to specifically?
